test/libsolidity/syntaxTests/largeTypes/storage_parameter.sol

```diff
@@ -3,4 +3,4 @@ contract C {
     function f(S storage) internal {}
 }
 // ----
-// Warning 2332: (64-65): Type "struct C.S" covers a large part of storage and thus makes collisions likely. Either use mappings or dynamic arrays and allow their size to be increased only in small quantities per transaction.
+// Warning 7325: (64-65): Type struct C.S covers a large part of storage and thus makes collisions likely. Either use mappings or dynamic arrays and allow their size to be increased only in small quantities per transaction.
```

Comment:
I think the quotes were useful. It reduces ambiguity in case of structs and enums with weird names.

Reply:
I agree that quotes could be useful, but we do not add them in other places, e.g.

```
"Expected contract but got " +
actualTypeType->actualType()->toString(true) +
"."

"Type " + varType->toString(true) + " is only valid in storage because it contains a (nested) mapping."
```

If we value consistency, we could add quotes to all such messages at once (I would do it in a separate PR though).

Reply:
You have a point. I thought quotes were used a bit more consistently :)

I'd still leave them here since they were already there but if you prefer to remove them then it's fine too.